core/primitives/src/lib.rs

```diff
@@ -26,11 +26,17 @@
 /// Can be used to create a `HashMap`.
 #[macro_export]
 macro_rules! map {
-	($( $name:expr => $value:expr ),*) => (
+    ($( $name:expr => $value:expr ),* ,) => (
```

Comment:
can use ` $(,)?` to handle optional trailing comma

Reply:
Oh, I forget the syntax.